### PR TITLE
The weighted FFT filter must recalculate the filter each time

### PR DESCRIPTION
**Pull request by isullivan**
Addresses bug issue #180

### CODE REVIEW

**Comment by rlbyrne** [comment]
Unfortunately this branch didn't fix the issue. The plots look the same as before.

**Comment by rlbyrne** [comment]
The issue seems to be resolved by calculating a per-polarization renormalization (filling) factor, rather than calculated one across polarizations. See Issue #180 for details. @isullivan I'm recommending that these changes get merged, can you please review?

File: fhd_output/get_image_renormalization.pro [hunk 1]
```diff
@@ -9,14 +9,16 @@ dimension=obs.dimension
...
 FOR pol_i=0,n_pol-1 DO BEGIN    
-    normalization_arr[pol_i]=1./(dirty_image_generate(*weights_arr[pol_i],degpix=degpix,obs=obs,psf=psf,params=params,$
+    renorm_factor[pol_i]=1./(dirty_image_generate(*weights_arr[pol_i],degpix=degpix,obs=obs,psf=psf,params=params,$
         weights=*weights_arr[pol_i],pad_uv_image=pad_uv_image,image_filter_fn=image_filter_fn,$
         filter=filter_arr[pol_i],antialias=antialias,beam_ptr=beam_base[pol_i],_Extra=extra))[dimension/2.,elements/2.]
-    normalization_arr[pol_i]*=((*beam_base[pol_i])[obs.obsx,obs.obsy])^2.
+    renorm_factor[pol_i]*=((*beam_base[pol_i])[obs.obsx,obs.obsy])^2.
 ENDFOR
-renorm_factor=mean(normalization_arr[0:n_pol-1])
+IF keyword_set(image_filter_fn) THEN IF image_filter_fn NE 'filter_uv_weighted' THEN BEGIN
```
Comment (isullivan) [inline fhd_output/get_image_renormalization.pro]: Change this to:
`IF filter_name NE 'weighted' THEN BEGIN`

File: fhd_output/get_image_renormalization.pro [hunk 2]
```diff
@@ -9,14 +9,16 @@ dimension=obs.dimension
...
+    renorm_factor[*]=mean(renorm_factor[0:n_pol-1])
```
Comment (isullivan) [inline fhd_output/get_image_renormalization.pro]: I know it was there before, but drop the `[0:n_pol-1]`

Comment (isullivan) [reply fhd_output/get_image_renormalization.pro]: And you could combine those two changes to replace the whole block with:
`IF filter_name NE 'weighted' THEN renorm_factor[*]=mean(renorm_factor)`

**Pull request merged**